ecommerce/mail_api.py

```diff
@@ -87,7 +87,9 @@ def send_ecommerce_refund_message(order_record):
                 recipient,
                 {
                     "order": order_record,
-                    "readable_id": line.purchased_object.readable_id if line else None,
+                    "readable_id": line.purchased_object.readable_id
+                    if line and len(line.purchased_object.readable_id.split("+")) < 2
+                    else line.purchased_object.readable_id.split("+")[1],
```

Comment:
Does it make sense to use the existing property course_number from the Course model? something like line.purchased_object.course_number

Reply:
Yeah this is a good idea!  It will work well for the CourseRun model, but I don't think it can be used for the Course model.

Reply:
Good find!

Reply:
@rachellougee I also added the course_number property to the Course model since it previously only existed on the Course_Run model (even though that just referenced the Course model).